e2e/capacity/provider_boost.test.ts

```diff
@@ -53,4 +62,140 @@ describe('Capacity: provider_boost extrinsic', function () {
+    await ExtrinsicHelper.runToBlock(await getNextRewardEraBlock());
+    await ExtrinsicHelper.runToBlock(await getNextRewardEraBlock());
+    await ExtrinsicHelper.runToBlock(await getNextRewardEraBlock());
+
+    const bobBalance = await getFreeBalance(bob);
+    const aliceBalance = await getFreeBalance(alice);
+
+    const innerClaim = ExtrinsicHelper.api.tx.capacity.claimStakingRewards();
+    const expectedClaimEvent = ExtrinsicHelper.api.events.capacity.ProviderBoostRewardClaimed;
+    await ExtrinsicHelper.proxySignAndSend(innerClaim, alice, bob, expectedClaimEvent);
+
+    // Verify that alice's proxy call cost balance and alice account does not get rewards.
+    const aliceNewBalance = await getFreeBalance(alice);
+    assert(
+      aliceNewBalance < aliceBalance,
+      `expected alice not to get claimed rewards. Alice new - old = ${aliceNewBalance - aliceBalance}`
+    );
+
+    // assert that bob got the claimed rewards.
+    const bobNewBalance = await getFreeBalance(bob);
+    assert(bobNewBalance > bobBalance, `expected bob to get rewards. Bob new - old = ${bobNewBalance - bobBalance}`);
+
+    const innerUnstake = ExtrinsicHelper.api.tx.capacity.unstake(1, boostAmount);
+    const expectedUnstakeEvent = ExtrinsicHelper.api.events.capacity.UnStaked;
+    await ExtrinsicHelper.proxySignAndSend(innerUnstake, alice, bob, expectedUnstakeEvent);
+  });
+
+  it('multisig call as a proxy works for capacity staking ops', async function () {
+    // Input the addresses that will make up the multisig account.
+    const alice = await createAndFundKeypair(fundingSource, 2000n * DOLLARS, 'alice');
+    const bob = await createAndFundKeypair(fundingSource, 2000n * DOLLARS, 'bob');
+    const charlie = await createAndFundKeypair(fundingSource, 2000n * DOLLARS, 'charlie');
+    const fergie = await createAndFundKeypair(fundingSource, 2000n * DOLLARS, 'fergie');
+
+    const stakeKeys = createKeys('booster');
+    const provider = await createMsaAndProvider(fundingSource, stakeKeys, 'Provider1', providerBalance);
+
+    const owners = [getUnifiedAddress(alice), getUnifiedAddress(bob), getUnifiedAddress(charlie)];
+
+    // The number of accounts that must approve. Must be greater than 0 and less than
+    // or equal to the total number of addresses.
+    const threshold = 2;
+
+    // Convert byte array to SS58 encoding.
+    const ss58MultiAddress = encodeMultiAddress(owners, threshold);
+
+    await addProxy(fergie, ss58MultiAddress, 'Any');
+    const api = ExtrinsicHelper.api;
+
+    // 2) Build inner call: Fergie -> transfer to Bob
+    const boostAmount = 2n * DOLLARS;
+    const inner = api.tx.capacity.providerBoost(provider, boostAmount);
+
+    // 3) Wrap with proxy: multisig will act as Fergie
+    const proxyCall = api.tx.proxy.proxy(
+      getUnifiedAddress(fergie), // real
+      null, // forceProxyType = None (use Any granted above)
+      inner
+    );
+
+    // Helpers
+    // the weight for this is pretty high
```

Comment:
I pulled the weights from the Frequency txn that failed and bumped them up a bit